packages/graphback-runtime-knex/src/knexQueryMapper.ts

```diff
@@ -101,7 +101,7 @@ function where(builder: Knex.QueryBuilder, filter: any, or: boolean = false, not
 
     // eslint-disable-next-line no-null/no-null
     if (exprEntry[1] === null) {
-      builder = builder[builderMethod('whereNull', or, not)](col)
+      builder = builder[builderMethod('whereNull', or, exprEntry[0] === 'ne')](col)
```

Comment:
Is this changed related?

Reply:
I found a bug while testing the querying.

Reply:
Nice catch. What was it?

Reply:
`ne` was being evaluated as `eq`, so {ne: null} in SQL gave `IS NULL`.

Added https://github.com/aerogear/graphback/pull/1574/commits/0ac97c2d25774a468f08e243c67d81dbfdab712b to cover it.

Reply:
Amazing thanks.